submission/programs/adae.R

```diff
@@ -0,0 +1,283 @@
+# Name: ADAE
```

Comment:
Include header. We should agree on a standard one.

Reply:
Used same header from adsl.R in main.  Now updated.

Reply:
@SHAESEN2 or @thomas-neitmann If ok, please `Resolve conversation`.